config/environments/development.rb

```diff
@@ -10,6 +10,10 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = true
 
+  # Whitelist one hostname
+  puts "Adding conjur machine to whitelist"
+  config.hosts << "conjur"
+
```

Comment:
What's going on with this setting in production configuration

Reply:
I did not anticipate a problem with this new Rails "feature" in appliance configuration.

`config/environments/appliance.rb` configuration, which was not modified, is loaded in appliance tests and its all green there.

I'm checking why.